edgecast/waf/access_rule.go

```diff
@@ -119,3 +128,56 @@ func (svc *WAFService) GetAccessRulesLight(accountNumber string) ([]AccessRuleLi
 
 	return *accessRuleLight, nil
 }
+
+//AccessRuleLightById containts detail of rules that identify traffic for access control
+type AccessRuleLightById struct {
```

Comment:
A lot of these properties overlap with AccessRule - remove them here and use an embedded `AccessRule` like we demo'd earlier today

Reply:
In addition, I think `AccessRuleDetail` is a more appropriate name here.